### DISCUSSION TITLE
Missing Keyboard Features From AC (2023 Strix G17 G713PI)

### DISCUSSION THREAD

**Discussion by aspire-a** [seerge/g-helper#2818] [Ideas]
### Is it possible to add some of the features from the Armoury Create to G-Helper.
1. Setting to turn on/off bottom light strip
2. Setting to turn on/off laptop startup light sequence
3. Setting to turn on/off light sequence while sleep
4. Setting to adjust direction for the rainbow mode
5. Setting to apply random color for some of the keyboard modes
(In another discussions it is mentioned that modes don't support custom colors by default and it is sent through AC which costs a lot of resource to run. But it would be nice to have it then people can choose if they want to enable it or not)

### Armoury Create Screenshots for the mentioned settings 
Armoury Create is in Turkish. Couldn't find a easy way to change the language but added translations next to the Turkish equivalents annotated with (*)

**Lightning Mode Rainbow Direction Setting.** 
I don't think this is done through AC because after uninstalling AC it continued to wave through colors in my selected direction. Direction set back to default (Left to Right) when I installed G-Helper.
![ac feature](https://github.com/seerge/g-helper/assets/71707804/6e4ad2da-f687-483f-a01c-2b311513ef1f)

**More Settings**
![ac settings](https://github.com/seerge/g-helper/assets/71707804/db0fd722-1940-419c-8fdf-87e753c4dc8f)

**Start, Close:** Toggles startup/shutdown light sequence
**Wakeup:** Completely turns on/off lights
**Sleep:** Toggles sleep light sequence (I guess it only works while charging)
Others allows you to adjust their(keyboard and bottom light strip) behavior on battery or while charging.

### Device
Rog Strix G17 2023 G713PI, Ryzen 9 7845HX, RTX4070

**Comment by seerge** [comment]
@aspire-a hello, you can do all this pretty much already

> 1. Setting to turn on/off bottom light strip
> 2. Setting to turn on/off laptop startup light sequence
> 3. Setting to turn on/off light sequence while sleep

Is possible via ``Extra`` settings

> 5. Setting to apply random color for some of the keyboard modes
(In another discussions it is mentioned that modes don't support custom colors by default and it is sent through AC which costs a lot of resource to run. But it would be nice to have it then people can choose if they want to enable it or not)

It is possible if you will use last build from https://github.com/seerge/g-helper/discussions/2807  -> [GHelper.zip](https://github.com/user-attachments/files/16117525/GHelper.zip)

It will be live in next release

The only thing you can't customize atm is direction of a rainbow, and mainly cause i don't want to overcomplicate UI with even more extra toggles :)

**Reply by aspire-a** [reply]
Thank you for answering this quickly. I just saw the options I don't know how I didn't realized that they exist. Also downloaded the last build and it works thank you.  It would be nice to have the direction choice, maybe in future. Thanks again.
